include/laik/backend.h

```diff
@@ -24,16 +24,8 @@
 // Do not include this file in LAIK applications!
 //
 
-#ifndef _LAIK_INTERNAL_H_
-#error "include laik-internal.h instead"
-#endif
-
-// "laik-internal.h" includes all the following. This is just to make IDE happy
-#include "core.h"
-#include "space.h"
-#include "space-internal.h"
-#include "data-internal.h"
-#include <stdbool.h>
+#include <laik.h>     // for Laik_Transition, Laik_Data, Laik_Instance, Laik...
```

Comment:
```backend.h``` is not part of the public API (since it is not included by ```laik.h```). Therefore it can make use of the public API (i.e. include ```laik.h```) without causing a circular include.